src/containers/Bookings.js

```diff
@@ -1,19 +1,23 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import React, { Component } from "react";
+import Search from "../components/Search.js";
+import Results from "../components/Results.js";
+import FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
+  state = { reservations: [] };
+
   search = () => {
-    console.info('TO DO!');
+    console.info("to do");
+    this.setState({ reservations: FakeBookings });
```

Comment:
Well done implementing state to show the bookings! We'll cover state in more detail in the class

Reply:
Now that we are actually using state to update the results, I think this is slightly incorrect. If you type something in the search input, it will filter the results correctly, but if you then click the search button, it will reset the results back to showing everything. I don't think that is the intended functionality.

However, I've realised that we're going to cover a better way of handling forms/input in the class today (called *refs*) that will hopefully make this easier to do.